src/render.tsx

```diff
@@ -1,16 +1,19 @@
 import TestRenderer from 'react-test-renderer';
 import type { ReactTestInstance, ReactTestRenderer } from 'react-test-renderer';
 import * as React from 'react';
+import { Profiler } from 'react';
 import act from './act';
 import { addToCleanupQueue } from './cleanup';
 import debugShallow from './helpers/debugShallow';
 import debugDeep from './helpers/debugDeep';
 import { getQueriesForElement } from './within';
-import { setRenderResult } from './screen';
+import { setRenderResult, screen } from './screen';
+import { assertStringsWithinText } from './helpers/assertStringsWithinText';
 
 export type RenderOptions = {
   wrapper?: React.ComponentType<any>;
   createNodeMock?: (element: React.ReactElement) => any;
+  unstable_validateStringsRenderedInText?: boolean;
```

Comment:
Seems like React Natives uses `within` in this context for the original error message: `Invariant Violation: Text strings must be rendered within a <Text> component`

```suggestion
  unstable_validateStringsRenderedWithinText?: boolean;
```